### ISSUE TITLE
hash character in smiles string

### ISSUE THREAD

**Issue by cailing20** [ropensci/webchem#280]
I noticed some of my smiles queries had returned NA and they have hash character (triple bond) in the smiles string.
```r
erlotinib.smiles <- "COCCOC1=CC2=C(C=C1OCCOC)C(NC1=CC=CC(=C1)C#C)=NC=N2"
is.smiles(erlotinib.smiles) # TRUE
get_cid(erlotinib.smiles, from = "smiles") # NA as cid
```
When the hash character is replaced by "%23" (as a way to excape it from URLencode), it works
```r
erlotinib.smiles<-"COCCOC1=CC2=C(C=C1OCCOC)C(NC1=CC=CC(=C1)C%23C)=NC=N2"
is.smiles(erlotinib.smiles) # FALSE
get_cid(erlotinib.smiles, from = "smiles") # 176870 as cid
```
It took me a while to figure this out. I think it might be good to check whether there is any hash characters and if so automatically repace with "%23" in the "get_cid" function when the argument "from" is specified as "smiles". Thank you.

**Comment by Aariq** [comment]
Thanks for pointing this out.  We have fixed this bug in the development version, you can download it with `remotes::install_github("ropensci/webchem"), or use the `URLencode()` workaround like you discovered.

Duplicate of #269, fixed in PR #270

**Comment by cailing20** [comment]
Thank you Eric! Sorry I missed the previous issue.
Ling
